iranian_cities/mixins/base_location.py

```diff
@@ -0,0 +1,14 @@
+from django.db.models import Model, CharField, BigIntegerField
+from django.utils.translation import gettext_lazy as _
+
+
+class BaseLocation(Model):
+    name = CharField(verbose_name=_("Name"), max_length=255)
```

Comment:
- [ ] Each field must have `help_text` and `db_comment` attributes.

- [ ] `help_text` must be userfriendly for end user and `db_comment` is for developer that knows terminologies.

- [ ] So, don't forget to make `help_text` translatable

Reply:
@sepehr-akbarzadeh, thank you for your valuable feedback! I've implemented the changes based on your suggestions. Please review the updated code and let me know if there are any further adjustments needed.